rules/data/embedded/embedded-svg.yara

```diff
@@ -0,0 +1,44 @@
+rule svg: medium {
+  meta:
+    description = "Contains SVG file(s)"
+
+  strings:
+    $svg_open  = "<svg>" nocase
+    $svg_close = "</svg>" nocase
```

Comment:
avoid `nocase` when possible - https://github.com/Neo23x0/YARA-Performance-Guidelines

Basically unless we've seen folks trying to evade detection via case sensitivity, I'd add both a caps & lowercase example.

Reply:
Updated in [`00ddb63` (#704)](https://github.com/chainguard-dev/malcontent/pull/704/commits/00ddb6315da42f1d23b32da5a91293492291a916).